SU2_CFD/include/limiters/CLimiterDetails.hpp

```diff
@@ -87,6 +87,36 @@ struct LimiterHelpers
     Type factor = 0.5*(1.0+dist+sin(PI_NUMBER*dist)/PI_NUMBER);
     return max(0.0, min(factor, 1.0));
   }
+
+  FORCEINLINE static Type R3Function(const Type& proj, const Type& delta, const Type& epsp)
+  {
+    Type Dp = fabs(delta); Type Dm = fabs(proj);
+    if(Dp>(2.0*Dm)) {return 1.0;}
+    else {
+        Type y = Dp*Dp*Dp + epsp;
+        Type S3 = 4.0*Dm*Dm;
+        return ((y + Dp*S3) / (y + Dm*(delta*delta+S3)));}
+  }
```

Comment:
For consistency with the other functions in this file:
```suggestion
  FORCEINLINE static Type r3Function(const Type& proj, const Type& delta, const Type& epsp)
  {
    Type Dp = fabs(delta);
    Type Dm = fabs(proj);
    if(Dp>(2.0*Dm)) return 1.0;
    Type y = pow(Dp, 3) + epsp;
    Type S3 = 4.0*Dm*Dm;
    return (y + Dp*S3) / (y + Dm*(delta*delta+S3));
  }
```